docs/source/projects.rst

```diff
@@ -368,108 +368,8 @@ Run an MLflow Project on Databricks
-simpler "new cluster specification" format. For example:
-
-  .. code-block:: json
-
-    {
-      "spark_version": "6.4.x-scala2.11",
-      "node_type_id": "i3.xlarge",
-      "aws_attributes": {"availability": "ON_DEMAND"},
-      "num_workers": 4
-    }
-
-If you do need to install libraries on the worker, use the "cluster specification" format. For example:
-
-  .. code-block:: json
-
-    {
-      "new_cluster": {
-        "spark_version": "6.4.x-scala2.11",
-        "node_type_id": "i3.xlarge",
-        "aws_attributes": {"availability": "ON_DEMAND"},
-        "num_workers": 4
-      },
-      "libraries": [
-        {
-          "pypi": {
-            "package": "tensorflow"
-          }
-        },
-        {
-          "whl": "dbfs:/path_to_my_lib.whl"
-        }
-      ]
-    }
-
-If you need those same libraries available on the driver, you will also need to specify them in the
-Conda environment YAML file. You can reference wheels on DBFS using the ``pip`` section of the environment
-file. For example:
-
-  .. code-block:: yaml
-  
-    dependencies:
-      - tensorflow
-      - pip:
-        - /dbfs/path_to_ml_lib.whl
-
-The path to the wheel is different in the Conda environment YAML file than the JSON specification used
-to install libraries on the Spark workers. This is because Conda does not support DBFS; you must instead
-use the `DBFS FUSE mount <https://docs.databricks.com/data/databricks-file-system.html#local-file-apis>`_
-to read the file as if it were a local file.
-
-2. Run your project using the following command:
-
-  .. code-block:: bash
-
-    mlflow run <project_uri> -b databricks --backend-config <json-new-cluster-spec>
-
-  where ``<project_uri>`` is a Git repository URI or a folder.
-
-.. important::
-
-  - Databricks execution for MLflow projects with Docker environments is *not* currently supported.
-
-  - You must use a *new cluster* specification when running an MLflow Project on Databricks. Running
-    Projects against existing clusters is not currently supported.
-
-Databricks Execution Tips
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When running an MLflow Project on Databricks, the following tips may be helpful.
-
-Using SparkR on Databricks
-##########################
-
-In order to use SparkR in an MLflow Project run on Databricks, your project code must first install
-and import SparkR as follows:
-
-.. code-block:: R
-
-  if (file.exists("/databricks/spark/R/pkg")) {
-    install.packages("/databricks/spark/R/pkg", repos = NULL)
-  } else {
-    install.packages("SparkR")
-  }
-
-  library(SparkR)
-
-Your project code can then proceed to initialize a SparkR session and use SparkR as normal:
-
-.. code-block:: R
-
-  sparkR.session()
-  ...
+`Databricks CLI <https://github.com/databricks/databricks-cli>`_. Find detailed instructions
+in the ` Databricks docs <https://docs.databricks.com/applications/mlflow/index.html>`_.
```

Comment:
Nice! We should also still keep both the Azure and AWS doc links - in general, we don't to link folks on Azure to AWS docs

Reply:
![image](https://user-images.githubusercontent.com/51172624/91216195-688bdc00-e6ca-11ea-967b-2838607f2a55.png)

Didn't they link to the same thing?

Reply:
Ah, you're right they did 😱 - can we change to https://docs.microsoft.com/en-us/azure/databricks/applications/mlflow/projects

Reply:
Discussed offline: they shouldn't have :)